graphql/util.go

```diff
@@ -674,13 +674,13 @@ type VersionModifications struct {
 	Active   bool                      `json:"active"`
 	Abort    bool                      `json:"abort"`
 	Priority int64                     `json:"priority"`
-	TaskIds  []string                  `json:"task_ids"`
+	TaskIds  []*model.TaskToRestart    `json:"task_ids"`
 }
 
 func ModifyVersion(version model.Version, user user.DBUser, proj *model.ProjectRef, modifications VersionModifications) (int, error) {
 	switch modifications.Action {
 	case Restart:
```

Comment:
This switch case is called into from the Legacy UI Restart Modal and changing the type for TaskIds will break it: https://github.com/evergreen-ci/evergreen/blob/main/public/static/js/version_admin.js#L50 

In order to retain backwards compatibility with Spruce and the Legacy UI, it's probably best to make an entirely new switch statement here called `RestartVersions` and remove `case Restart` once the Legacy UI and Spruce UI no longer rely on it.

Reply:
I added a check. Does that work?